client/src/util/createClient.tsx

```diff
@@ -0,0 +1,27 @@
+import { InMemoryCache } from "apollo-cache-inmemory";
+import ApolloClient from "apollo-client";
+import { ApolloLink } from "apollo-link";
+import { BatchHttpLink } from "apollo-link-batch-http";
+import { createPersistedQueryLink } from "apollo-link-persisted-queries";
+import get from "lodash/get";
+
+export default function createClient(uri: string) {
```

Comment:
There is nothing in this file indicating that this is a graphql-client, maybe rename file and function to createGraphqlClient?

Reply:
Good point. Renamed